RELEASE-NOTES-1.39

```diff
@@ -5,12 +5,61 @@ PHP 8.1 workboard: https://phabricator.wikimedia.org/tag/php_8.1_support/
 PHP 8.2 workboard: https://phabricator.wikimedia.org/tag/php_8.2_support/
 PHP 8.3 workboard: https://phabricator.wikimedia.org/tag/php_8.3_support/
 
-== MediaWiki 1.39.7 ==
+== MediaWiki 1.39.8 ==
```

Comment:
nit: we should update PR title

Reply:
nah, 1.39.8 just started and this is 1.39.7 update